crates/extensions/tedge_downloader_ext/src/actor.rs

```diff
@@ -41,6 +41,7 @@ impl DownloadRequest {
         }
     }
 
+    #[deprecated = "Setting ownership is a privileged operation, only tedge-write which runs as root should do this"]
     pub fn with_permission(self, permission: PermissionEntry) -> Self {
```

Comment:
Why not simply removing this method? This crate is only internally used.

Reply:
Amended the commit to remove the method.